.config/dotnet-tools.json

```diff
@@ -15,10 +15,10 @@
       ]
     },
     "fake-cli": {
-      "version": "5.21.0",
+      "version": "1.0.0-1",
```

Comment:
Can you please revert this change

Reply:
I rebased onto release/next.
Is it acceptable to you ?